rbac/internal/urls.py

```diff
@@ -19,13 +19,29 @@
 
 from django.urls import path
 
-from . import views
+from . import integration_views, views
 from .views import trigger_error
 
 urlpatterns = [
     path("api/tenant/unmodified/", views.list_unmodified_tenants),
     path("api/tenant/", views.list_tenants),
     path("api/tenant/<str:tenant_name>/", views.tenant_view),
+    path("api/tenant/<str:account_number>/groups/", integration_views.groups, name="integration-groups"),
```

Comment:
This all looks good. We probably could just use org_id now for those new endpoints. Something along the lines of:
`api/tenant/<str:org_id>/groups/<str:uuid>/roles/`

Otherwise everything looks good!